native/src/components/screens/categoryScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import { RootNavigatorParamList } from '@components/navigation/RootNavigator'
+import { ChallengeCard } from '@components/ui/ChallengeCard'
+import { RouteProp } from '@react-navigation/native'
+import { Layout, Text } from '@ui-kitten/components'
+import _ from 'lodash'
+import React from 'react'
+import { Dimensions, ScrollView } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useChallengesByCategoryQuery } from '../../../graphqlSdk'
+
+const COLORS = {
+  orange: '#ffc342',
+  navy: '#0c2945',
+  cyan: '#37a2a4',
+}
+
+type Props = {
+  route: RouteProp<RootNavigatorParamList, 'Category'>
+}
+
+export const CategoryScreen: React.FC<Props> = ({ route }) => {
+  const { category } = route.params
+  const { data, loading, error } = useChallengesByCategoryQuery({
+    variables: { user_id: '8003885c-e560-4263-a4e1-171293278a50', category },
+  })
+  const challenges = data?.category_by_pk?.challenges
+
+  if (loading) {
+    return <Text>Loading...</Text>
+  }
+
+  if (error || !challenges) {
+    return <Text>Error!</Text>
+  }
+
+  const challengesWithColors = challenges.map(challenge => ({
+    ...challenge,
+    color: _.sample(COLORS),
+  }))
+
+  return (
+    <Layout style={{ flex: 1 }} level="1">
+      <Layout level="3">
+        <SafeAreaView>
+          <Text category="h3" style={{ marginLeft: 16, marginBottom: 16, marginTop: 32 }}>
+            {/* MILAN category by mal byt Pascalcase */}
```

Comment:
Sprav prosim niekde mapu z tych values co dostavame na to co mame zobrazovat a potom to mozme vsade reusnut.